src/wasm.rs

```diff
@@ -38,11 +39,13 @@ impl Provider for WasmRuntime {
             .clone()
             .namespace
             .unwrap_or_else(|| "default".into());
-        // Start with a hard-coded WASM file
-        let data = std::fs::read("./examples/greet.wasm")
-            .expect("greet.wasm should be in examples directory");
-        pod_status(client.clone(), pod.clone(), "Running", namespace.as_str());
+        let first_container = pod.spec.containers[0].clone();
+
+        let module_ref = pod.spec.containers[0].image.as_ref().unwrap().to_owned();
+        pull_wasm(module_ref, String::from("pulled.wasm")).unwrap();
```

Comment:
Use the real module name instead of `pulled.wasm`.

Reply:
For now we pull the module into `pulled.wasm`.
We don't have a central location for all pulled modules, and we always pull the module from a registry.

We should decide on a cache directory for modules.

Reply:
We can port the ImageStore crate over eventually for storing modules.